k8s/test/e2e/local-assembly-container/assembly.py

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env python3 # pylint: disable=missing-module-docstring
```

Comment:
We have already `pylint: disable=missing-module-docstring` in the pylintrc

Reply:
But in the `Codacy Static Code Analysis`, I think the pylintrc doesn't work?

Reply:
We have a lot python files.

If pylintrc doesn't work, we should fix pylintrc, rather than repeat the hack to every file.

Reply:
Yeah, but is there a way to update the configuration of `Codacy Static Code Analysis` to support pylintrc?